src/test/java/com/ibm/slsa/maven/plugin/ProvenanceGeneratorTest.java

```diff
@@ -52,31 +56,48 @@
 @ExtendWith(MockitoExtension.class)
 public class ProvenanceGeneratorTest {
 
-    @Mock private MavenProject project;
-    @Mock private Build projectBuild;
-    @Mock private MavenSession mavenSession;
-    @Mock private Log log;
+    @Mock
+    private MavenProject project;
+    @Mock
+    private Build projectBuild;
+    @Mock
+    private MavenSession mavenSession;
+    @Mock
+    private Log log;
 
     private String builderId = "myBuilderId";
     private String buildType = "myBuildType";
     private CommonTestUtils testUtils = new CommonTestUtils();
 
     DefaultArtifactHandler artifactHandler = new DefaultArtifactHandler(Constants.PACKAGE_TYPE);
-    DefaultArtifact artifact = new DefaultArtifact("com.example", builderId, "1.0", builderId, buildType, buildType, artifactHandler);
+    DefaultArtifact artifact = new DefaultArtifact("com.example", builderId, "1.0", builderId, buildType, buildType,
+            artifactHandler);
 
     @Test
     void test_generateProvenanceFileData() {
         ProvenanceGenerator generator = new ProvenanceGenerator(builderId, buildType, project, mavenSession, log);
 
-        when(project.getArtifact()).thenReturn(artifact);
-        when(project.getBuild()).thenReturn(projectBuild);
-        when(projectBuild.getDirectory()).thenReturn(Constants.RESOURCES_DIR + File.separator + "one-package");
-        when(mavenSession.getStartTime()).thenReturn(new Date());
-        when(project.getBuild().getFinalName()).thenReturn(Constants.FINAL_NAME_APP);
+        lenient().when(project.getArtifact()).thenReturn(artifact);
+        lenient().when(project.getBuild()).thenReturn(projectBuild);
+        lenient().when(projectBuild.getDirectory())
+                .thenReturn(Constants.RESOURCES_DIR + File.separator + "one-package");
+        lenient().when(mavenSession.getStartTime()).thenReturn(new Date());
+        lenient().when(project.getBuild().getFinalName()).thenReturn(Constants.FINAL_NAME_APP);
+
+        ProjectDependencyGraph pdg = mock(ProjectDependencyGraph.class);
+        lenient().when(mavenSession.getProjectDependencyGraph()).thenReturn(pdg);
+
+        MavenProject mp1 = createProjectParent();
+        MavenProject mp2 = createProjectChild1(mp1);
+        MavenProject mp3 = createProjectChild2(mp1);
+
+        List<MavenProject> theList = Arrays.asList(mp1, mp2, mp3);
+        lenient().when(pdg.getSortedProjects()).thenReturn(theList);
 
         try {
             JsonObject statement = generator.generateProvenanceFileData();
-            testUtils.assertJsonOnlyContainsKeys("Provenance data", statement, Statement.KEY_TYPE, Statement.KEY_SUBJECT, Statement.KEY_PREDICATE_TYPE, Statement.KEY_PREDICATE);
+            testUtils.assertJsonOnlyContainsKeys("Provenance data", statement, Statement.KEY_TYPE,
+                    Statement.KEY_SUBJECT, Statement.KEY_PREDICATE_TYPE, Statement.KEY_PREDICATE);
```

Comment:
These type of changes are just formatting changes and it would be good to exclude them from the PR. That just unneeded noise to the PR.